hudi-flink/src/main/java/org/apache/hudi/table/HoodieTableFactory.java

```diff
@@ -98,6 +102,33 @@ public String factoryIdentifier() {
   //  Utilities
   // -------------------------------------------------------------------------
 
+  /** Validate required options. e.g record key and pre combine key.
+   *
+   * @param conf The table options
+   * @param schema The table schema
+   */
+  private void validateRequiredOptions(Configuration conf, TableSchema schema) {
```

Comment:
sounds like `validateRequiredFields`? should we handle the partition field as well if it's a partitioned table?

Reply:
good, rename the method.
partitioned table needn't validate.it has default partiton value.we just verify hoodie required fields.